### PR TITLE
fix(chat): refactor Quote component, fix reactivity

### PR DESCRIPTION
**Pull request by Antreesy**
### ☑️ Resolves

* Follow-up https://github.com/nextcloud/spreed/pull/12395#discussion_r1618771274

### 🏁 Checklist

- [x] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [x] 🖥️ Tested with Desktop client or should not be risky for it 
- [x] ⛑️ Tests are included or not possible
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/composables/useMessageInfo.js [hunk 1]
```diff
@@ -17,19 +17,13 @@ const canEditMessage = getCapabilities()?.spreed?.features?.includes('edit-messa
...
- * @param {import('vue').Ref} tokenRef conversation token
- * @param {import('vue').Ref} messageIdRef message id to edit
- * @return {import('vue').ComputedRef<boolean>}
+ * @param {import('vue').Ref} message message object
+ * @return {Record<string, import('vue').ComputedRef<boolean>>}
  */
-export function useMessageInfo(tokenRef = ref(null), messageIdRef = ref(null)) {
-	const token = unref(tokenRef)
-	const messageId = unref(messageIdRef)
+export function useMessageInfo(message = ref({})) {
```
Comment (ShGKme) [inline src/composables/useMessageInfo.js:23]: I don't think, the default value, especially an empty object which is not a special case for a message object, makes sense here.

`if (...!message.value)` also doesn’t work for `useMessageInfo()` with `{}` value.

Comment (ShGKme) [reply src/composables/useMessageInfo.js:23]: > Maybe, check for any key present in object?

We can check for `id` property only.

But in a follow-up (not right now) I'd refactor it and replace this `{}` value with explicit `null`.

**Review by ShGKme — APPROVED** [review]
Thanks!

**Review by DorraJaouad — APPROVED** [review]
tested

**Pull request merged**